Engine/src/Engine/ResourceManagement/ShaderGenerator.h

```diff
@@ -1,17 +1,18 @@
 #pragma once
 #ifndef SHADERGENERATOR_H
 #define SHADERGENERATOR_H
+
+#include <Engine/ResourceManagement/ResourceManager.h>
```

Comment:
this include is still unused

Reply:
Needed to bring in the GLenum. Otherwise system crashes.

Reply:
Same as above

Reply:
Done